java/src/main/native/src/jni_compiled_expr.hpp

```diff
@@ -7,7 +7,7 @@
  *
  *     http://www.apache.org/licenses/LICENSE-2.0
  *
- * Unless required by applicable law or agreed to in writing, software
+ *expression Unless required by applicable law or agreed to in writing, software
```

Comment:
Looks like an errant paste?
```suggestion
 * Unless required by applicable law or agreed to in writing, software
```